changelog.d/2420.misc.rst

```diff
@@ -0,0 +1 @@
+Replace Python 3.9.0 beta with 3.9.0 final on GitHub Actions.
```

Comment:
FWIW I'm trying to make the change notes explanation more illustrative in another PR: https://github.com/pypa/setuptools/pull/2395/files#diff-b0a8c7744649eb7f19a8eb9ccee0eb7bR1.